ingestors/arena-ingestor/src/main/kotlin/no/nav/amt/tiltak/ingestors/arena/processors/TiltaksgjennomforingProcessor.kt

```diff
@@ -35,6 +35,12 @@ open class TiltaksgjennomforingProcessor(
 
 			val tiltaksleverandor = addTiltaksleverandor(newFields)
 
+			if (tiltaksleverandor == null) {
```

Comment:
Ser ikke konteksten utenfor fila her, men kunne man sjekket `newFields.ARBGIV_ID_ARRANGOR` før man forsøker å legge til leverandør? Ville isåfall framstå som mer eksplisitt enn "noe" i `addTiltaksleverandor` som kan returnere null i forskjellige tilfeller

Reply:
Yeah, det blir litt cleanere